torch/csrc/api/src/nn/init.cpp

```diff
@@ -146,6 +165,33 @@ Tensor uniform_(Tensor tensor, double low, double high) {
   return tensor.uniform_(low, high);
 }
 
+Tensor kaiming_uniform_(Tensor tensor, double a, FanMode mode, Nonlinearity nonlinearity) {
+  Fan fan(tensor);
```

Comment:
You would have to add a `NoGradGuard` here.

Reply:
Yes that's important